src/components/elements/SkipToContentButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button } from '@mui/material';
+import { ReactNode } from 'react';
+
+type SkipToContentProps = {
+  focusTargetId: string;
+  children?: ReactNode;
+};
+
+const SkipToContent: React.FC<SkipToContentProps> = ({
+  focusTargetId,
+  children,
+}) => {
+  const handleClick = () => {
+    const container = document.getElementById(focusTargetId);
```

Comment:
There's a tradeoff here with putting focus on the container itself, rather than trying to find the first focusable element in the container. The latter strategy gives a better UX for keyboard navigation, but for voiceover, it skips content that should be included.

The keyboard navigation UX problem is that setting focus to the container sometimes leads to focused content getting scrolled out of view, but I _think_ that it's still worth the tradeoff for better VO experience. Let me know what you think.

To reproduce: Open a long form, keyboard nav to the "skip to content button", and enter. The form gets focus. Continuing to hit tab navigates the form correctly, but the elements you're navigating are scrolled out of view. (Observed on Chrome)

Reply:
It looks like using `container.focus({ preventScroll: true });` can prevent this issue. Not sure if that's the desirable behavior in all cases though - it looks fine on assessments in chrome.

Another option is to scroll, but the top offset will depend on where we are. For assessments, this looks ok:
```
      container.focus();
      scrollToElement(container, STICKY_BAR_HEIGHT + CONTEXT_HEADER_HEIGHT);
```

Reply:
Thanks, cool! I think I prefer `preventScroll: true`. It seems simpler to reason about: In skip-to-content navigation situations, we always want to keep scroll at the top of the page. (Rather than reasoning through the different scroll offsets in different contexts within the app.)